examples/website-demos/virtual-event/src/components/CentralPanel/CentralPanel.scss

```diff
@@ -0,0 +1,8 @@
+@import '../../styles/variables';
+
+.central-panel-container {
+  height: 100%;
+  width: 100%;
+  //   background: $primary-accent;
```

Comment:
why this commented out?

Reply:
I dont know if I will need that background for the next two components in the center panel

Reply:
if it's in the design then add it just in case